chromogenic/version.py

```diff
@@ -5,7 +5,7 @@
 from subprocess import Popen, PIPE
 from os.path import abspath, dirname
 
-VERSION = (0, 4, 17, 'dev', 0)
+VERSION = (0, 5, 0, 'dev', 1)
```

Comment:
Should we just drop this, and do `(0, 5, 0)`. What is the remainder there for?

Reply:
Remainder is just there while I hacked on it. If you think it looks good then we can strip it.